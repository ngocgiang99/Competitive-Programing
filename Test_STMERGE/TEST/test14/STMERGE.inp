2
1 6
745281198 601598909 168562508 522661828 952252734 238539600 
8 3
153601773 821086541 233594428 
88726197 765455617 149409932 
983885809 600819031 660239616 
460956703 965028330 641683697 
671892142 5288285 963813390 
328536312 930737168 884432858 
33055359 733546785 195274091 
743857004 695268773 849638752 
