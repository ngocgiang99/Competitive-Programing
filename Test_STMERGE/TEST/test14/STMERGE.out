238539600
233594428
